### PR TITLE
Add Icon set for Oracle Cloud Services. 

### PR DESCRIPTION
**Pull request by McSly**
- Add icon set for Oracle Cloud
- Update python scripts to handle the new provider
- Update documentation
- Update autogen.sh to use Image Magic instead of Inkscape to crop and resize and save the icons as PNG files as the current version of Inkscape is broken and can't export file to png

### CODE REVIEW

File: scripts/resource.py [hunk 1]
```diff
@@ -57,11 +57,22 @@ def cleaner_k8s(f):
     return f.lower()
 
 
+def cleaner_oci(f):
+    f = f.replace("-", "_")
```
Comment (mingrammer) [inline scripts/resource.py]: diagrams uses hyphen-case for image resource names. Please keep the consistency :)

File: docs/guides/node.md [hunk 2]
```diff
@@ -44,6 +44,12 @@ from diagrams.gcp.ml import AutoML
 from diagrams.k8s.compute import Pod, StatefulSet
 from diagrams.k8s.network import Service
 from diagrams.k8s.storage import PV, PVC, StorageClass
+...
+
+# Oracle resources
```
Comment (mingrammer) [inline docs/guides/node.md]: Use lowercase for this comment. `oracle resources`

File: README.md [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
 **Diagram as Code**.
 
-Diagrams lets you draw the cloud system architecture **in Python code**. It was born for **prototyping** a new system architecture design without any design tools. You can also describe or visualize the existing system architecture as well. Diagrams currently supports four major providers: `AWS`, `Azure`, `GCP` and `Kubernetes`.
+Diagrams lets you draw the cloud system architecture **in Python code**. It was born for **prototyping** a new system architecture design without any design tools. You can also describe or visualize the existing system architecture as well. Diagrams currently supports five major providers: `AWS`, `Azure`, `GCP`, `Oracle Cloud` and `Kubernetes`.
```
Comment (mingrammer) [inline README.md]: `alibabscloud` provider was added. You need to update it!

ref: https://github.com/mingrammer/diagrams/pull/19

**Comment by mingrammer** [comment]
\+ Please resolve the conflicts

**Comment by McSly** [comment]
Changes requested have been made

**Comment by mingrammer** [comment]
`__init__.py` is missing for OCI. Please add this file to `diagrams/oci/__init__.py`

```python
"""
OCI provides a set of services for Oracle Cloud provider.
"""

from diagrams import Node


class _OCI(Node):
    _provider = "oci"
    _icon_dir = "resources/oci"

    fontcolor = "#ffffff"

```

**Comment by mingrammer** [comment]
Thank you for contribution :)

**Pull request merged**